spec/models/event_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Event, model: true do
```

Comment:
Two things here:

* You don't need to use `Rspec` in front of describe
* Because you have `config.infer_spec_type_from_file_location!`, you don't need to specify the type, i.e. `model: true`